src/gui/creds/shibboleth/shibbolethwebview.cpp

```diff
@@ -61,6 +61,8 @@ ShibbolethWebView::ShibbolethWebView(AccountPtr account, QWidget* parent)
     setAttribute(Qt::WA_DeleteOnClose);
 
     QWebPage* page = new UserAgentWebPage(this);
+    page->setNetworkAccessManager( account->networkAccessManager() );
```

Comment:
Sorry 👎 
This is a too critical change for a patch release.
As we re-create the QNAM in some other code path I don't know how crashy this could be